src/msw/listctrl.cpp

```diff
@@ -3249,6 +3252,8 @@ void wxListCtrl::OnPaint(wxPaintEvent& event)
     const bool drawHRules = HasFlag(wxLC_HRULES);
     const bool drawVRules = HasFlag(wxLC_VRULES);
 
+    static int iRefresh = 0;
```

Comment:
This really shouldn't be necessary and if it is, it needs a big comment explaining why is it so.

Reply:
The purpose of the variable aside: Why is it shared by all existing `wxListCtrl` instances?